src/app/client/src/app/modules/core/services/framework/framework.service.ts

```diff
@@ -0,0 +1,135 @@
+import { Injectable } from '@angular/core';
+import { UserService } from './../user/user.service';
+import { ContentService } from './../content/content.service';
+import {
+  ConfigService, ToasterService, ResourceService, ServerResponse,
+  Framework, FrameworkCategorie, IUserData, IUserProfile
+} from '@sunbird/shared';
+import { SearchParam } from '@sunbird/core';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
```

Comment:
i think no need to add map and throw operators explicitly, they will be available with Observable. Check and update me